### DISCUSSION TITLE
Why is wordpress child theme not recommended to be used in _tw?

### DISCUSSION THREAD

**Discussion by aboutjquery** [gregsullivan/_tw#151] [Q&A]
Are there some simple examples of why?

I always want my theme code to be up to date. Possible reasons for this are security, design or better syntax, etc., which may be corrected when the theme is updated.

So I want to use a child theme, is this possible? If so, are there some simple examples to get started? Thanks

**Comment by gregsullivan** [comment]
Hello again!

Just to set up some context, could you please take a look at these two pages:

* [Keeping your theme up-to-date](https://underscoretw.com/docs/updating/)
* [Don’t Update Your Theme (From _s)](https://themeshaper.com/2013/10/11/dont-update-your-theme/)

If you still have questions after reading both of those posts, please let me know and I'll do my best to answer!
